### DISCUSSION TITLE
Pi5 with SSD, OLED and Battery!

### DISCUSSION THREAD

**Discussion by ruebyi** [outdoorbits/little-backup-box#325] [Show and tell]
Hi, 

from the day I got hold on my first LBB I wished it would be portable - that's why I wanted the LBB with the pi Zero 2.

With some Inspiration from the discussions here I started with my pi5 Project.

It's build around a pi5 with Waveshare OLED with Joystick and 3 Buttons
Geekworm X1200 USP and Geekworm X1001 for SSD

The case is missing, but a friend of mine is printing one for me, so I hope I can show you the whole LBB in a few days.
![IMG_8911](https://github.com/user-attachments/assets/4a2beb72-584b-462c-8cbf-ee57088e7bef)
![IMG_8912](https://github.com/user-attachments/assets/371d4eaa-2c9f-44bc-ace6-23698cf82adc)
![IMG_8913](https://github.com/user-attachments/assets/02f8eca6-9e9b-4468-91ee-5cf015639c33)
![IMG_8914](https://github.com/user-attachments/assets/e938381f-217a-4e8a-8790-d9a08d717b04)
![IMG_8907](https://github.com/user-attachments/assets/d0ec6193-f2e5-4f25-96ee-7dee23848fe6)

**Comment by AeolusBNW** [comment]
full parts list?

**Reply by ruebyi** [reply]
Pi 5
Geekworm X1200 USP 
Geekworm X1001 for SSD
Crucial 2TB P3 Plus
Waveshare 1.3inch OLED Display Hat 128x64 Pixels with Embedded Controller
2x Berry Base Stacking Header 6,2mm 
2x 18650 Battery

I don't know wich Spacers I used, but I can Update on this.

**Reply by AeolusBNW** [reply]
Awesome, thank you very much.
